model/articles.model.js

```diff
@@ -4,7 +4,7 @@ exports.fetchAllArticles = () => {
   return db
     .query(
       `SELECT articles.*,CAST(COUNT(comments.article_id) AS INTEGER) AS comment_count FROM comments
-     JOIN articles ON articles.article_id=comments.article_id
+  FULL OUTER JOIN articles ON articles.article_id=comments.article_id
```

Comment:
good catch! 

inner join would only return records that exist in **both** databases

This can be a `LEFT JOIN`!